src/Avalonia.Controls/GroupBox.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Avalonia.Controls.Primitives;
+using Avalonia.Input;
+
+namespace Avalonia.Controls
+{
+    /// <summary>
+    /// GroupBox Control class
+    /// </summary>
+    public class GroupBox: HeaderedContentControl
+    {
+        /// <summary>
+        /// style key of this control
+        /// </summary>
+        public Type StyleKey => typeof(GroupBox);
```

Comment:
This shouldn't be needed

Reply:
Ok. I though we have to add this line. Perhabs I misunderstood something.

Reply:
only if you class is of a different type to the template...

if you class was MyCustomGroupBox, you can use this to make it get the template GroupBox has.